### DISCUSSION TITLE
ArgoCD 2.12.0 has been released but not tested in kubernetes

### DISCUSSION THREAD

**Discussion by HenryXie1** [argoproj/argo-cd#19506] [General]
I noticed that ArgoCD 2.12.0 has been released. However, according to the [documentation](https://argo-cd.readthedocs.io/en/stable/operator-manual/installation/#tested-versions), it has not been tested in kubernetes yet.

Could we get some updates on this?

**Comment by crenshaw-dev** [comment]
The automation that's supposed to update that matrix is apparently broken. 2.12 has been tested on Kubernetes 1.27-1.30 for quite a while. You should be fine with any of those versions.
